deployment-aws/terraform/apiserver.tf

```diff
@@ -165,6 +167,9 @@ resource "aws_instance" "api" {
   export FACTER_EMAIL_HOST_PASSWORD="${aws_iam_access_key.ses_user.ses_smtp_password_v4}"
   export FACTER_FROM_EMAIL='${var.from_email}'
   export FACTER_FRONTEND_DOMAIN='${var.frontend_domain}'
+  export FACTER_GLOBUS_APP_CLIENT_ID='${var.globus_app_client_uuid}'
+  export FACTER_GLOBUS_APP_CLIENT_SECRET='${var.globus_app_client_secret}'
+  export FACTER_GLOBUS_ENDPOINT_ID='${var.globus_endpiont_id}'
```

Comment:
From `terraform apply`:

An input variable with the name "globus_endpiont_id" has not been declared. Did you mean "globus_endpoint_id"?

Reply:
good catch. I added that after and never ran apply